Modules/Core/Common/src/itkMetaDataDictionary.cxx

```diff
@@ -182,9 +191,32 @@ void
 MetaDataDictionary
 ::Clear()
 {
+  MakeUnique();
```

Comment:
Will this create a copy, only to be erased by the next line?

Reply:
I’ll look into that